### PR TITLE
Update interprocess.md

### PR DESCRIPTION
**Pull request by JamesNK**
Mention more advantages of IPC and put them into a list.

Plus some minor clean up.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/grpc/interprocess.md](https://github.com/dotnet/AspNetCore.Docs/blob/2f9547037a3b4267b084ac68869edf492714267d/aspnetcore/grpc/interprocess.md) | [Inter-process communication with gRPC](https://review.learn.microsoft.com/en-us/aspnet/core/grpc/interprocess?branch=pr-en-us-30310) |

### CODE REVIEW

**Review by wadepickett — APPROVED** [review]
Looks good!  I suggested a very minor change, just a topic meta data ms.date update and a contraction.

File: aspnetcore/grpc/interprocess.md [hunk 1]
```diff
@@ -38,15 +38,19 @@ The preceding project file:
 
 ## Inter-process communication (IPC) transports
 
-gRPC calls between a client and server on different machines are usually sent over TCP sockets. TCP is a great choice for communicating across a network or the Internet. However, IPC transports can offer performance advantages and better integration with OS features when the client and server are on the same machine.
+gRPC calls between a client and server on different machines are usually sent over TCP sockets. TCP is a great choice for communicating across a network or the Internet. However, IPC transports offer advantages when communicating between processes on the same machine:
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/interprocess.md:41]:
```suggestion
gRPC calls between a client and server on different machines are usually sent over TCP sockets. TCP is a good choice for communicating across a network or the Internet. However, IPC transports offer advantages when communicating between processes on the same machine:
```

Comment (Rick-Anderson) [reply aspnetcore/grpc/interprocess.md:41]: Technical document, not marketing doc.

File: aspnetcore/grpc/interprocess.md [hunk 2]
```diff
@@ -38,15 +38,19 @@ The preceding project file:
...
 
 * [Unix domain sockets (UDS)](https://wikipedia.org/wiki/Unix_domain_socket) is a widely supported IPC technology. UDS is the best choice for building cross-platform apps, and it's usable on Linux, macOS, and [Windows 10/Windows Server 2019 or later](https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/).
-* [Named pipes](https://wikipedia.org/wiki/Named_pipe) are supported by all versions of Windows. Named pipes integrate well with [Windows security](/windows/win32/ipc/named-pipe-security-and-access-rights), which can be used to control client access to the pipe.
-* Other IPC transports by implementing <xref:Microsoft.AspNetCore.Connections.IConnectionListenerFactory> and registering the implementation at app startup.
+* [Named pipes](https://wikipedia.org/wiki/Named_pipe) are supported by all versions of Windows. Named pipes integrate well with [Windows security](/windows/win32/ipc/named-pipe-security-and-access-rights), which can control client access to the pipe.
+* Support other IPC transports by implementing <xref:Microsoft.AspNetCore.Connections.IConnectionListenerFactory> and registering the implementation at app startup.
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/interprocess.md:51]:
```suggestion
* Additional IPC transports by implementing <xref:Microsoft.AspNetCore.Connections.IConnectionListenerFactory> and registering the implementation at app startup.
```

Comment (Rick-Anderson) [reply aspnetcore/grpc/interprocess.md:51]: Not really happy with that last bullet. Is there a better adjective than Additional. Something more like custom.

File: aspnetcore/grpc/interprocess.md [hunk 3]
```diff
@@ -38,15 +38,19 @@ The preceding project file:
...
 
-Cross-platform apps may want to use different IPC transports, depending on the current OS. An app can check the current operating system on startup and chose the desired transport for that platform:
+Depending on the current OS, cross-platform apps may want to use different IPC transports. An app can check the current operating system on startup and choose the desired transport for that platform:
```
Comment (Rick-Anderson) [inline aspnetcore/grpc/interprocess.md:53]: I get why you're using current, so you can determine the OS at run time. But I don't think current is needed. Our mantra, less is more. [Use bigger ideas, fewer words](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#use-bigger-ideas-fewer-words) : Our modern design hinges on crisp minimalism. Shorter is always better.
[Short sentences and fragments are easier to scan and read.](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human#a-focus-on-the-customer) Prune every excess word.

Comment (Rick-Anderson) [inline aspnetcore/grpc/interprocess.md:53]:
```suggestion
Depending on the OS, cross-platform apps may choose different IPC transports. An app can check the OS on startup and choose the desired transport for that platform:
```

**Review by Rick-Anderson — APPROVED** [review]
Your version is fine, consider my suggestions.

File: aspnetcore/grpc/interprocess.md (hunk 3 above)
Comment (JamesNK) [reply aspnetcore/grpc/interprocess.md:53]:
```suggestion
Depending on the OS, cross-platform apps may want to use different IPC transports. An app can check the operating system on startup and choose the desired transport for that platform:
```

**Pull request merged**